qr-code/src/main/java/org/odk/collect/qrcode/QRCodeDecoder.kt

```diff
@@ -26,14 +27,26 @@ class QRCodeDecoderImpl : QRCodeDecoder {
         return try {
             val bitmap = BitmapFactory.decodeStream(inputStream)
 
-            val decodedQrCode = QRCodeMultiReader()
-                .decode(
-                    getBinaryBitmap(bitmap),
-                    mapOf(
-                        DecodeHintType.TRY_HARDER to TRUE,
-                        DecodeHintType.POSSIBLE_FORMATS to BarcodeFormat.QR_CODE
+            val decodedQrCode = try {
+                QRCodeMultiReader()
+                    .decode(
+                        getBinaryBitmap(bitmap),
+                        mapOf(
+                            DecodeHintType.TRY_HARDER to TRUE,
+                            DecodeHintType.POSSIBLE_FORMATS to BarcodeFormat.QR_CODE
+                        )
                     )
-                )
+            } catch (e: NotFoundException) {
+                QRCodeMultiReader()
```

Comment:
It's probably worth adding a comment here describing what's going on.

Reply:
Done.